1225-my_institution/360-institution_administrator_opens_my_institution.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Institution Administrator opens My Institution
+
+    @360
+    Scenario: Institution Administrator opens My Institution
+        Given that the user is logged in as Institution Administrator
+        When they click the menu item My Institution
+        Then they see the My Institution page
+        And they see fields:
+            | Name in organization registry |
+            | Display name                  |
+            | Short display name            |
+            | Archive name                  |
+        And they can click Save to persist values in fields
```

Comment:
Er det nødvendig å ta med `to persist values in fields`? mener vi har skrevet `And they see the Save button` i tidligere scenario. Dette scenarioet handler jo ikke om å lagre noe.